test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -156,9 +182,129 @@ def test_set_and_unset_sched_attributes(self):
         self.server.manager(MGR_CMD_LIST,
                             SCHED,
                             id="TestCommonSched")
-        sched = None
-        sched = self.server.schedulers['TestCommonSched']
-        self.assertNotEqual(sched, None)
-        self.assertEqual(
-            sched.attributes['sched_cycle_length'],
-            '00:20:00')
+        a = {'sched_cycle_length': '00:20:00'}
+        self.server.expect(SCHED, a, id='TestCommonSched', max_attempts=10)
+
+    def test_sched_default_attrs(self):
+        """
+        Test all sched attributes are set by default on default scheduler
+        """
+        sched_priv = os.path.join(
+            self.server.pbs_conf['PBS_HOME'], 'sched_priv')
+        sched_logs = os.path.join(
+            self.server.pbs_conf['PBS_HOME'], 'sched_logs')
+        a = {'sched_port': 15004,
+             'sched_host': self.server.hostname,
+             'sched_priv': sched_priv,
+             'sched_log': sched_logs,
+             'scheduling': 'True',
+             'scheduler_iteration': 600,
+             'state': 'idle',
+             'sched_cycle_length': '00:20:00'}
+        self.server.expect(SCHED, a, id='default', max_attempts=10)
```

Comment:
When you pass more than one attribute to expect() it does an or.  If you want an and, do op=PTL_AND